src/graphs/two_sat.rs

```diff
@@ -29,16 +28,6 @@ impl TwoSat {
         }
     }
 
-    /// Add a new variable
-    ///
-    /// # Complexity
-    /// - Time: O(1)
-    /// - Space: O(1)
-    pub fn add_var(&mut self) {
```

Comment:
code coverage marked this as an uncalled function (not call from anywhere in examples/*)

Reply:
it's called in the doc test, but true we should add it to the test

Reply:
can you add that as part of this PR?

Reply:
Shouldn't the coverage be failing then?